src/filter/mod.rs

```diff
@@ -13,9 +13,11 @@ use util;
 mod programmable_filter;
 pub mod delay_filter;
 mod flush_boundary_filter;
+mod json_encode_filter;
 
 pub use self::delay_filter::{DelayFilter, DelayFilterConfig};
 pub use self::flush_boundary_filter::{FlushBoundaryFilter, FlushBoundaryFilterConfig};
+pub use self::json_encode_filter::{JSONEncodeFilter, JSONEncodeFilterConfig};
```

Comment:
These are now exposed but you'll need to add them to `src/config.rs` to hook up configuration.